priv/user_linux.go

```diff
@@ -120,20 +118,16 @@ func RunAs(uid, gid int) error {
 	}
 	_ = runtime.GOMAXPROCS(mxp)
 
-	if err := os.Setenv("HOME", user.HomeDir); err != nil {
-		return err
-	}
-	if err = os.Setenv("USER", user.Name); err != nil {
-		return err
-	}
-	if _, ok := os.LookupEnv("DOCKER_CONFIG"); ok {
+	if !withUserLookup {
 		return nil
 	}
-	// ggcr sets default docker config during init, fix for user
-	if err := os.Setenv("DOCKER_CONFIG", filepath.Join(user.HomeDir, ".docker")); err != nil {
+
+	foundUser, err := user.LookupId(strconv.Itoa(uid))
```

Comment:
Can we move this into `setEnvironmentForUser`, export that function, and call it directly from `createCmd.Privileges`? That way we could take the bool out of the `RunAs` signature. I think that usage would read a little more obviously/nicely in the lifecycle commands. The lookup is a impl. detail, the env changes are the behavior.

Reply:
Sounds good, I will make the change